File: fizzbuzz.py
#counts from 1 to 100, outputting "Fizz" for multiples of 3, "Buzz" for multiples of 5, and "FizzBuzz" for multiples of both

for i in range(1,101):
    output = ""
    if i % 3 == 0:
        output += "Fizz"   
    if i % 5 == 0:
        output += "Buzz"
    
    if output == "":
        print i
    else:
        print output

File: fizzbuzz.c
/*
A program that counts from 1 to 100
For multiples of three, outputs "Fizz"
For multiples of five, outputs "Buzz"
For multiples of both three and five, outputs "FizzBuzz"
*/
#include <stdio.h>
#include <stdlib.h>

const int mult1 = 3;
const int mult2 = 5;

int main() {
    for (int i = 1; i <= 100; i++)
    {
        if (i % mult1 == 0 && i % mult2 == 0) 
        {
            printf("FizzBuzz\n");
        }
        else if (i % mult1 == 0)
        {
            printf("Fizz\n");
        }
        else if (i % mult2 == 0)
        {
            printf("Buzz\n");
        }
        else 
        {
            printf("%d\n",i);
        }
    }
    
    return 0;
}